tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/04_checkInvoiceDownloadedFromList.js

```diff
@@ -0,0 +1,149 @@
+      this.pageObjects = await init();
+    });
+  });
+
+  describe('Check invoice file in BO', async () => {
+    it('should go to the orders page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToOrdersPage', baseContext);
+      await this.pageObjects.boBasePage.goToSubMenu(
+        this.pageObjects.boBasePage.ordersParentLink,
+        this.pageObjects.boBasePage.ordersLink,
+      );
+      const pageTitle = await this.pageObjects.ordersPage.getPageTitle();
+      await expect(pageTitle).to.contains(this.pageObjects.ordersPage.pageTitle);
+    });
+
+    it(`should update order status to '${Statuses.paymentAccepted.status}'`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'updateOrderStatus', baseContext);
+      const textResult = await this.pageObjects.ordersPage.setOrderStatus(1, Statuses.paymentAccepted);
+      await expect(textResult).to.equal(this.pageObjects.ordersPage.successfulUpdateMessage);
+      const orderStatus = await this.pageObjects.ordersPage.getTextColumn('osname', 1);
+      await expect(orderStatus, 'Order status was not updated').to.equal(Statuses.paymentAccepted.status);
+    });
+
+    it('should download invoice', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'downloadInvoice', baseContext);
+      await this.pageObjects.ordersPage.downloadInvoice(1);
+      const fileExist = await files.checkFileExistence('IN', 5000, true, '.pdf');
+      await expect(fileExist).to.be.true;
+    });
+
+    it('should check invoice pdf text', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkInvoiceText', baseContext);
+      invoiceFilename = await files.getFileNameFromDir(global.BO.DOWNLOAD_PATH, 'IN', '.pdf');
+      const orderInformation = await this.pageObjects.ordersPage.getOrderFromTable(1);
+      // Check Reference in pdf
+      const referenceExist = await files.checkTextInPDF(invoiceFilename, orderInformation.reference);
+      await expect(referenceExist, `Reference '${orderInformation.reference}' does not exist in invoice`).to.be.true;
+      // Check country name in delivery Address in pdf
+      const deliveryExist = await files.checkTextInPDF(invoiceFilename, orderInformation.delivery);
+      await expect(deliveryExist, `Country name '${orderInformation.delivery}' does not exist in invoice`).to.be.true;
+      // Check customer name in pdf
+      const customerExist = await files.checkTextInPDF(invoiceFilename, orderInformation.customer.slice(3));
+      await expect(customerExist, `Country name '${orderInformation.customer}' does not exist in invoice`).to.be.true;
+      // Check total paid in pdf
+      const totalPaidExist = await files.checkTextInPDF(invoiceFilename, orderInformation.totalPaid);
+      await expect(totalPaidExist, `Total paid '${orderInformation.totalPaid}' does not exist in invoice`).to.be.true;
```

Comment:
We won't have the old problem with the return to next line ?

Reply:
I run `npm run lint` before pushing the PR, so i don't think so